examples/networking-role/roles/system/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+dns_servers:
```

Comment:
Shouldn't this be in defaults and not vars? This is something a user would commonly override correct?